api/src/routes/team.js

```diff
@@ -74,4 +74,28 @@ teamRouter.patch(
   teamController.updateTeam,
 );
 
+teamRouter.delete(
+  '/:team_id',
+  authMiddleware.cookieJwtAuth,
+  mongooseMiddleware.validateTeamObjectId,
+  teamMiddleware.validateTeamId,
+  teamController.deleteTeam,
+);
+
+teamRouter.delete(
+  '/:team_id/clear',
+  authMiddleware.cookieJwtAuth,
+  mongooseMiddleware.validateTeamObjectId,
+  teamMiddleware.validateTeamId,
+  teamController.clearTeam,
+);
+
+teamRouter.delete(
+  '/:team_id/deleteGHPages/:code',
```

Comment:
if code is meant to be the token, probably don't pass it in the URL, pass it in the body instead